### PR TITLE
Resolves #447

### LINKED ISSUE #447: How to retrieve encryption method from powershell
Hi,

Is there any way to get Get-BitlockerVolume to display the encryption method?
Specifically if hardware or software encryption is being used?

Valentijn

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7f3b0c94-181c-f2d8-70b0-d65c46f1c92b
* Version Independent ID: 9bae55ce-a087-0878-8bb6-5edee5f9d33b
* Content: [Get-BitLockerVolume (bitlocker)](https://docs.microsoft.com/en-us/powershell/module/bitlocker/get-bitlockervolume?view=win10-ps)
* Content Source: [docset/windows/bitlocker/get-bitlockervolume.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/bitlocker/get-bitlockervolume.md)
* Product: **w10**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

### PR DESCRIPTION
**Pull request by velkovb**
Added example showing the EncryptionMethod info. Added link to the article mapping numbers to encryption methods. Updated Bitlocker overview article and fixed the description section. Updates done in all versions of the article.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/447

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/bitlocker/get-bitlockervolume.md [hunk 1]
```diff
@@ -37,24 +37,19 @@ If you do not specify a drive letter, this cmdlet gets all volumes for the curre
...
+- Mount Point - Drive letter.
+- CapacityGB - Size of drive.
+- VolumeStatus - Whether BitLocker currently protects some, all, or none of the data on the volume. 
+- Encryption Percentage - Percent of the volume protected by BitLocker.
+- KeyProtector - Type of key protector or protectors.
+- AutoUnlock Enabled - Whether BitLocker uses automatic unlocking for the volume.
+- Protection Status - Whether BitLocker currently uses a key protector to encrypt the volume encryption key.
+- EncryptionMethod - Indicates the encryption algorithm and key size used on the volume.
+
+For an overview of BitLocker, see [BitLocker Overview](https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-overview)
```
Comment (nenonix) [inline docset/windows/bitlocker/get-bitlockervolume.md]:
```suggestion
For an overview of BitLocker, see [BitLocker Overview](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-overview)
```

File: docset/windows/bitlocker/get-bitlockervolume.md [hunk 2]
```diff
@@ -37,24 +37,19 @@ If you do not specify a drive letter, this cmdlet gets all volumes for the curre
...
+
+For an overview of Encryption methods, see [GetEncryptionMethod method](https://docs.microsoft.com/en-us/windows/win32/secprov/getencryptionmethod-win32-encryptablevolume)
```
Comment (nenonix) [inline docset/windows/bitlocker/get-bitlockervolume.md]:
```suggestion
For an overview of Encryption methods, see [GetEncryptionMethod method](https://docs.microsoft.com/windows/win32/secprov/getencryptionmethod-win32-encryptablevolume).
```

File: docset/winserver2012-ps/bitlocker/Get-BitLockerVolume.md [hunk 3]
```diff
@@ -17,55 +17,44 @@ Gets information about volumes that BitLocker can protect.
...
+- Mount Point - Drive letter.
+- CapacityGB - Size of drive.
+- VolumeStatus - Whether BitLocker currently protects some, all, or none of the data on the volume. 
+- Encryption Percentage - Percent of the volume protected by BitLocker.
+- KeyProtector - Type of key protector or protectors.
+- AutoUnlock Enabled - Whether BitLocker uses automatic unlocking for the volume.
+- Protection Status - Whether BitLocker currently uses a key protector to encrypt the volume encryption key.
+- EncryptionMethod - Indicates the encryption algorithm and key size used on the volume.
+
+For an overview of BitLocker, see [BitLocker Overview](https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-overview)
```
Comment (nenonix) [inline docset/winserver2012-ps/bitlocker/Get-BitLockerVolume.md]:
```suggestion
See [BitLocker Overview](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-overview) for more information.
```

File: docset/winserver2012-ps/bitlocker/Get-BitLockerVolume.md [hunk 4]
```diff
@@ -17,55 +17,44 @@ Gets information about volumes that BitLocker can protect.
...
+
+For an overview of Encryption methods, see [GetEncryptionMethod method](https://docs.microsoft.com/en-us/windows/win32/secprov/getencryptionmethod-win32-encryptablevolume)
```
Comment (nenonix) [inline docset/winserver2012-ps/bitlocker/Get-BitLockerVolume.md]: For an overview of encryption methods, see [GetEncryptionMethod method](https://docs.microsoft.com/windows/win32/secprov/getencryptionmethod-win32-encryptablevolume).
```

File: docset/winserver2012r2-ps/bitlocker/Get-BitLockerVolume.md [hunk 5]
```diff
@@ -33,33 +33,29 @@ The **Get-BitLockerVolume** cmdlet gets information about volumes that BitLocker
...
+- Mount Point - Drive letter.
+- CapacityGB - Size of drive.
+- VolumeStatus - Whether BitLocker currently protects some, all, or none of the data on the volume. 
+- Encryption Percentage - Percent of the volume protected by BitLocker.
+- KeyProtector - Type of key protector or protectors.
+- AutoUnlock Enabled - Whether BitLocker uses automatic unlocking for the volume.
+- Protection Status - Whether BitLocker currently uses a key protector to encrypt the volume encryption key.
+- EncryptionMethod - Indicates the encryption algorithm and key size used on the volume.
+
+For an overview of BitLocker, see [BitLocker Overview](https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-overview)
```
Comment (nenonix) [inline docset/winserver2012r2-ps/bitlocker/Get-BitLockerVolume.md]:
```suggestion
See [BitLocker Overview](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-overview) for more information.
```

File: docset/winserver2012r2-ps/bitlocker/Get-BitLockerVolume.md [hunk 6]
```diff
@@ -33,33 +33,29 @@ The **Get-BitLockerVolume** cmdlet gets information about volumes that BitLocker
...
+
+For an overview of Encryption methods, see [GetEncryptionMethod method](https://docs.microsoft.com/en-us/windows/win32/secprov/getencryptionmethod-win32-encryptablevolume)
```
Comment (nenonix) [inline docset/winserver2012r2-ps/bitlocker/Get-BitLockerVolume.md]:
```suggestion
For an overview of encryption methods, see [GetEncryptionMethod method](https://docs.microsoft.com/windows/win32/secprov/getencryptionmethod-win32-encryptablevolume).
```

File: docset/winserver2012-ps/bitlocker/Get-BitLockerVolume.md (hunk 4 above)
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/bitlocker/Get-BitLockerVolume.md]:
```suggestion
For an overview of encryption methods, see [GetEncryptionMethod method](https://docs.microsoft.com/windows/win32/secprov/getencryptionmethod-win32-encryptablevolume).
```

File: docset/winserver2012r2-ps/bitlocker/Get-BitLockerVolume.md [hunk 7]
```diff
@@ -83,6 +80,27 @@ Data            E:        928.83 FullyDecrypted         0          {}
...
+ProtectionStatus     : On
+LockStatus           : Unlocked
+EncryptionPercentage : 100
+WipePercentage       : 0
+VolumeType           : OperatingSystem
+CapacityGB           : 218,2344
+KeyProtector         : {RecoveryPassword, Tpm}
+```
+
+This command lists all Bitlocker related attributes for C drive.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/bitlocker/Get-BitLockerVolume.md]:
```suggestion
This command lists all BitLocker related attributes for C drive.
```

**Comment by velkovb** [comment]
Thanks for the review. I have merged the changes.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by get-itips** [comment]
Looks ok!

**Comment by dariomws306** [comment]
This article looks much better now, good work!

I think we can remove brackets here:
PS C:\> (Get-BitLockerVolume -MountPoint C) | Format-List

**Comment by e0i** [comment]
@AndreaBarr The PR can be merged. Thanks.

**Pull request merged**